lightstep/constants.py

```diff
@@ -24,3 +24,7 @@
 
 # utils constants
 SECONDS_TO_MICRO = 1000000
+
+# Recorder constants
+flagMaxLogMessageLen = 1024
```

Comment:
(and the `flag` one is not actually a "flag" in the command-line sense, so we can remove that prefix)